components/graph-line.vue

```diff
@@ -6,14 +6,18 @@
       'graph-line--collapsed': isCollapsed,
     }"
   >
-    <ui-button-icon
+    <UiButtonIcon
```

Comment:
Please note that figure is a [sectioning root](https://html.spec.whatwg.org/#sectioning-root)